frontend/src/metabase/visualizations/components/ChartSettingsWidgetPopover.tsx

```diff
@@ -61,7 +61,7 @@ const ChartSettingsWidgetPopover = ({
                   name: sectionName,
                   value: sectionName,
                 }))}
-                onChange={setCurrentSection}
+                onChange={section => setCurrentSection(String(section))}
```

Comment:
Unfortunately when you `style` a generic component the type inference gets broken.